gbpservice/neutron/nsf/proxy/proxy.py

```diff
@@ -0,0 +1,145 @@
+import select
```

Comment:
This file is not part of config_agent ,removed core and proxy from config_agent_modules branch